apps/storybook/src/SelectToZoom.stories.tsx

```diff
@@ -1,39 +1,54 @@
 import {
-  DataCurve,
   Pan,
   VisCanvas,
   Zoom,
   SelectToZoom,
   ResetZoomButton,
+  HeatmapMesh,
+  getDomain,
 } from '@h5web/lib';
 import type { ModifierKey as ModifierKeyType } from '@h5web/lib';
+import { mockValues, ScaleType } from '@h5web/shared';
 import type { Meta, Story } from '@storybook/react';
-import { range } from 'lodash';
+import ndarray from 'ndarray';
 
 import FillHeight from './decorators/FillHeight';
 
-const X = range(-5, 5, 0.01);
-const Y = X.map((x) => Math.exp((-x * x) / 0.2));
+const cols = 41;
+const rows = 20;
+const values = ndarray(Float32Array.from(mockValues.twoD.flat()), [rows, cols]);
+const domain = getDomain(values);
 
-function GaussianCurve() {
-  return <DataCurve abscissas={X} ordinates={Y} color="blue" />;
+function Image() {
+  return (
+    <HeatmapMesh
+      values={values}
+      domain={domain || [0, 1]}
+      colorMap="Viridis"
+      scaleType={ScaleType.Linear}
+    />
+  );
 }
```

Comment:
I now display an `Image` in the story rather than a curve to demonstrate the `keepRatio` behaviour